src/generators/minecraft-character-heads/MinecraftCharacterHeadsGenerator.res

```diff
@@ -20,196 +21,138 @@ let images: array<Generator.imageDef> = [
   {id: "Background", url: requireImage("Background")},
   {id: "Tabs", url: requireImage("Tabs")},
   {id: "Folds", url: requireImage("Folds")},
+  {id: "Action Figure", url: requireImage("Action-Figure")},
 ]
 
-let textures: array<
-  Generator.textureDef,
-> = [/* {
+let textures: array<Generator.textureDef> = [
+  {
     id: "Skin 1",
     url: requireTexture("Steve"),
     standardWidth: 64,
     standardHeight: 64,
   },
   {
-    id: "Skin 2",
+    id: "Steve",
+    url: requireTexture("Steve"),
+    standardWidth: 64,
+    standardHeight: 64,
+  },
+  {
+    id: "Alex",
     url: requireTexture("Alex"),
     standardWidth: 64,
     standardHeight: 64,
   },
   {
-    id: "Skin 3",
+    id: "Zombie",
     url: requireTexture("Zombie"),
     standardWidth: 64,
     standardHeight: 64,
   },
   {
-    id: "Skin 4",
+    id: "Enderman",
     url: requireTexture("Enderman"),
     standardWidth: 64,
     standardHeight: 64,
   },
   {
-    id: "Skin 5",
+    id: "Skeleton",
     url: requireTexture("Skeleton"),
     standardWidth: 64,
     standardHeight: 64,
   },
   {
-    id: "Skin 6",
+    id: "Wither Skeleton",
     url: requireTexture("Wither_Skeleton"),
     standardWidth: 64,
     standardHeight: 64,
   },
   {
-    id: "Skin 7",
+    id: "Creeper",
     url: requireTexture("Creeper"),
     standardWidth: 64,
     standardHeight: 64,
   },
   {
-    id: "Skin 8",
+    id: "Blaze",
     url: requireTexture("Blaze"),
     standardWidth: 64,
     standardHeight: 64,
-  }, */]
+  },
+]
 
 let script = () => {
   // Define user inputs
   for i in 1 to 8 {
     Generator.defineTextureInput(
       "Skin " ++ Js.Int.toString(i),
-      {standardWidth: 64, standardHeight: 64, choices: []},
+      {
+        standardWidth: 64,
+        standardHeight: 64,
+        choices: [
```

Comment:
Great idea.

Reply:
> I noticed a glitch with the Alex model - can you take a look?

The Alex model needs to be switched to using the "Skin Model" dropdown, and that seems to be working fine. However, I was thinking of adding an "Auto" model that tested for pixels in the arm and changes the skin model to Steve or Alex accordingly.